models/mailchimp/mailchimp_members.sql

```diff
@@ -0,0 +1,3 @@
+select distinct
```

Comment:
same question. this table gets significantly bigger and i'm even more hesitant to want to do a distinct here unless we feel like it's absolutely necessary.